verifier/src/lib.rs

```diff
@@ -287,3 +317,13 @@ where
         .verify(&mut channel, &deep_evaluations, &query_positions)
         .map_err(VerifierError::FriVerificationFailed)
 }
+
+// ACCEPTABLE OPTIONS
+// ================================================================================================
+// Specifies either the minimal, conjectured or proven, security level or a set of
+// `ProofOptions` that are acceptable by the verification procedure.
+pub enum AcceptableOptions {
+    MinConjecturedSecurity(u32),
+    MinProvenSecurity(u32),
+    OptionSet(Vec<ProofOptions>),
```

Comment:
Since this is a part of the public API, we should add doc comments for each variant.

Also, to get rid of the errors in `no_std`, we should import `Vec` form `utils::collections`.